src/features/user/MFV2/ProfileCard/ProfileCardButton/index.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { ReactElement } from 'react';
+import styles from './ProfileCardButton.module.scss';
+import Link from 'next/link';
+
+interface CommonProps {
+  icon?: ReactElement;
+  text?: string;
+  type?: 'primary' | 'secondary';
+}
+
+interface LinkProps extends CommonProps {
+  href: string;
+  target?: '_blank' | '_self' | '_parent' | '_top';
+  isLink: 'true';
```

Comment:
I suggest calling this `elementType` with values of `link` and `button`. This makes the purpose clearer. You can keep a default of `button`

Also, the name `isLink` indicates a boolean value to me.